ci/request_pr_exp.py

```diff
@@ -196,6 +202,9 @@ def _parse_args(cmd) -> argparse.Namespace:
     args.request_memory = LARGE_REQUEST_MEM
     args.gke_template = LARGE_TEMPLATE_PATH
 
+  if (args.max_round == 100 and
+      any(args.name_suffix.startswith(suffix) for suffix in ['ascc-', 'dgk-'])):
+    args.max_round = 10
```

Comment:
I extended #809 to allow specifying the number of rounds directly via GitHub comments. I also reverted the default back to 100 rounds, as that’s typically needed for daily/weekly experiments.

This should not impact your PR experiment workflow. I understand you prefer setting the number of rounds to 10 and typically use the 'ascc-' prefix for the `--name-suffix`. To make this easier, I added a small trick to accommodate your workflow.